tutorials/backend/hasura/tutorial-site/content/data-modeling/2-try-user-queries.md

```diff
@@ -12,7 +12,7 @@ You can access GraphiQL by heading over to Console -> API -> GraphiQL tab.
 
 ## Mutation {#mutation}
 
-Let's add a user using a GraphQL Mutation. Copy the following code into the GraphiQL interface.
+Let's add a user using a [GraphQL Mutation](https://hasura.io/docs/latest/mutations/index/). Copy the following code into the GraphiQL interface.
```

Comment:
I would point users to https://hasura.io/learn/graphql/intro-graphql/graphql-mutations/